pkg/provisioner/ironic/ironic.go

```diff
@@ -70,6 +77,19 @@ func init() {
 		fmt.Fprintf(os.Stderr, "Cannot start: No IRONIC_INSPECTOR_ENDPOINT variable set")
 		os.Exit(1)
 	}
+	ironicCertFile = os.Getenv("IRONIC_CERT_FILE")
+	ironicKeyFile = os.Getenv("IRONIC_KEY_FILE")
+	ironicTrustedCAFile = os.Getenv("IRONIC_TRUSTED_CA_FILE")
+	if ironicCertFile != "" || ironicKeyFile != "" || ironicTrustedCAFile != "" {
+		if ironicCertFile != "" && ironicKeyFile != "" && ironicTrustedCAFile != "" {
+			ironicUseMutualTLS = true
+		} else {
+			fmt.Fprintf(os.Stderr, "Cannot start: IRONIC_CERT_FILE, IRONIC_KEY_FILE and IRONIC_TRUSTED_CA_FILE variables all must be set to use mTLS")
```

Comment:
This error message would be more helpful if it showed which variables are not set.